core/src/main/java/com/datastrato/gravitino/storage/relational/service/TableMetaService.java

```diff
@@ -21,7 +21,6 @@
 import java.util.Objects;
 import java.util.function.Function;
 
-/** The service class for table metadata. It provides the basic database operations for table. */
```

Comment:
Is this necessary?

Reply:
I also noticed this and Iplan to remove it.